test/models/ott/CategorisationInfoSpec.scala

```diff
@@ -536,10 +538,181 @@ class CategorisationInfoSpec extends SpecBase {
           longerCode = true
         )
 
-        val result = CategorisationInfo.build(ottResponse, "1234567890", longerCode = true)
+        val result =
+          CategorisationInfo.build(ottResponse, "1234567890", testTraderProfileResponseWithoutNiphl, longerCode = true)
         result.value mustEqual expectedResult
       }
 
+      ".NIPHL is authorised" - {
```

Comment:
Should perhaps be a section for testing Niphls cases whether or not user is authorised for Niphls

There should be a different answer if they don't have niphls but it is a niphls assessment (when with or without extra assessments)

Reply:
Tests updated